ci-operator/jobs/openshift/cluster-api-provider-openstack/openshift-cluster-api-provider-openstack-main-presubmits.yaml

```diff
@@ -0,0 +1,51 @@
+presubmits:
+  openshift/cluster-api-provider-openstack:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^main$
+    - ^main-
```

Comment:
Do we even plan to have `main-<something>` branches?

Reply:
If I remove that line, `make jobs` will add it back.

Reply:
Oh, okay.